### ISSUE TITLE
[Proposal] Replace `samplingType` and `observationType` with `sdo:additionalType`

### ISSUE THREAD

**Issue by ashleysommer** [ternaustralia/ontology_tern#223]
My understanding is `samplingType` and`observationType` were included in the TERN ontology to allow a kind of soft-typing mechanism, using more specific types from a vocabulary rather than adding new SubClasses to the TERN ontology.

There popular and increasing use of `sdo:additionalType` from schema.org that does the same thing, it is becoming the de-facto standard way of soft-typing across many data models.

The predicates `samplingType` and `observationType` in TERN ontology are not widely used, I don't know of any real world published datasets that use them. Most simply use/(abuse?) `usedProcedure` to classify different kinds of Samplings and Observations. 

I propose to remove `samplingType` and `observationType` from TERN ontology and replace them with `sdo:additionalType`.

Aside, this would be a good opportunity to remove `dcterms:type` profiling from the TERN ontology if it was an implementation detail and wasn't supposed to be in the published ontology.

**Comment by ashleysommer** [comment]
There remains a question about the use of `sdo:additionalType` on a `tern:Sample`.
Sample is a kind of a FeatureOfInterest, and TERN ontology uses `tern:featureType` to soft-type Features.
The correct thing to do here would be also replace featureType with `sdo:additionalType`, however `tern:featureType` is extensively used, and is hard to move away from.

**Comment by smguru** [comment]
Yes can be removed, I don't think we are using both properties. FYi @javiersan18 @junrongYu
